README.md

```diff
@@ -5,14 +5,17 @@
 
 ![Eddie Jaoude's LinkFree profile GIF](https://user-images.githubusercontent.com/82458069/145549448-b1feb5df-78aa-4286-a709-6985f0dade4c.gif)
 
-## Note for Hacktoberfest
-
-> A note for Hacktoberfest Participants:
->
-> Pull requests which add or edit your information in a `public/data/${yourname}.json` file will NOT be counted for Hacktoberfest.
->
-> Pull requests which improve the codebase, documentation, or other aspects of the project and are in line with the core values
-> of the event will count - maintainers will opt-in these PRs by applying the `hacktoberfest-accepted` label.
+<details close>
+  <summary> Note for Hacktoberfest </summary>
```

Comment:
I think we should add bold here or header.
```suggestion
  <summary><b> Note for Hacktoberfest </b></summary>
```

Reply:
I agree, @Panquesito7. It makes the dropdown between two images a bit more noticeable. I have implemented the same.